Documentation/api.md

```diff
@@ -202,7 +206,37 @@ The `NodeExporterCollectorCpufreqConfig` resource works as an on/off switch for
 
 | Property | Type | Description |
 | -------- | ---- | ----------- |
-| enabled | bool | A Boolean flag that enables or disables the `cpufreq` colletor. |
+| enabled | bool | A Boolean flag that enables or disables the `cpufreq` collector. |
+
+[Back to TOC](#table-of-contents)
+
+## NodeExporterCollectorNetDevConfig
+
+#### Description
+
+The `NodeExporterCollectorNetDevConfig` resource works as an on/off switch for the `netdev` collector of the `node-exporter` agent. By default, the `netdev` collector is enabled. If disabled, these metrics become unavailable: `node_network_receive_bytes_total`, `node_network_receive_compressed_total`, `node_network_receive_drop_total`, `node_network_receive_errs_total`, `node_network_receive_fifo_total`, `node_network_receive_frame_total`, `node_network_receive_multicast_total`, `node_network_receive_nohandler_total`, `node_network_receive_packets_total`, `node_network_transmit_bytes_total`, `node_network_transmit_carrier_total`, `node_network_transmit_colls_total`, `node_network_transmit_compressed_total`, `node_network_transmit_drop_total`, `node_network_transmit_errs_total`, `node_network_transmit_fifo_total`, `node_network_transmit_packets_total`.
```

Comment:
it seems `node_network_transmit_queue_length` is missing

Reply:
`node_network_transmit_queue_length` comes from netclass collector. 
Here is the code where this metric is generated.
https://github.com/prometheus/node_exporter/blob/7f05e312acd828f0c4ddcd194e275c05ffa35992/collector/netclass_linux.go#L129

Missing that metric may indicate an error in Node Exporter. During the test, has Node Exporter reported errors on netclass collector? The log lines look like `{timestamp} level=error msg="collector failed" name=netclass {details}`